src/cli.rs

```diff
@@ -1,78 +1,37 @@
-use structopt::StructOpt;
-
-/// Command line options
-#[derive(StructOpt, Debug)]
-#[structopt(name = "vtebench", about = "Benchmark Terminal Emulators")]
-pub struct Options {
-    #[structopt(short = "w", help = "width of terminal", default_value = "80")]
-    pub width: u16,
-
-    #[structopt(short = "h", help = "height of terminal", default_value = "24")]
-    pub height: u16,
+//! CLI parameter parsing.
```

Comment:
```suggestion
//! CLI arguments parsing.
```

Or the same with `parameters`.

Reply:
I don't thing singular is incorrect here? "parameters parsing" just sounds worse if you ask me.

Reply:
I mean that there're many arguments that the user could provide and you describe here a parsing for **all** of them. That's what my reasoning behind that. But I'm fine with singular, honestly.

Reply:
My idea is just that it parses each individual argument. Which is why I think both is fine and the current one just sounds better than the plural imho.

Reply:
Maybe we should rename it just to `argument` then, since I usually thing of those as arguments.

Reply:
Argument is fine too, but I also prefer the singular here. Args is definitely the more common name, since it's usually named as env::args/argv and stuff.

Reply:
Yeah, that's why I wrote it in a singular form here.

Reply:
English is weird. I'd write "CLI argument parsing" in the singular to mean that I'm parsing many arguments, but then I'd also write, "Parsed CLI arguments" to indicate that many arguments have been parsed. Perhaps there's some kind of rule that dictates this, but I could not tell you what that rule would be.